docs/tutorial/data.md

```diff
@@ -1,6 +1,7 @@
 # Data
 
 Up: [Contents](contents.md)
+
 Previous: [Draw functions](draw_functions.md)
```

Comment:
Yes, it's consistent now. The reason I did it is because it *does* affect formatting.
`master`:
![image](https://user-images.githubusercontent.com/12245827/153287707-1413fe82-0ec5-4b4c-ae8d-8d12a86d2ea7.png)

Mine:
![image](https://user-images.githubusercontent.com/12245827/153287797-586858f5-3a85-4d24-8075-89a1e50db74a.png)

Reply:
I said that, then I immediately noticed that `merging.md` wasn't consistent, whoops. Fixed now.

Reply:
Hmm, I suppose we're limited in what we can do by the markdown format. I don't think it was a bad thing that they took up less vertical space, but oh well.

Reply:
Well, we could theoretically do bullet points too:
![image](https://user-images.githubusercontent.com/12245827/153289281-b27e00ef-6e90-40d6-afc4-5f148b752199.png)

I'm fine with doing it on one line too; I just changed it because it looked unpolished and unintentional. If we want to do it on one line, that's totally fine. IMO, we just need some sort of separator between the two.

Reply:
Bullet points look nice.
But I think whichever option is fine, as long as it is consistent in the whole tutorial

Reply:
Let's just standardize on bullet points for now.